src/test/reverselock_tests.cpp

```diff
@@ -7,7 +7,9 @@
 
 #include <boost/test/unit_test.hpp>
 
-BOOST_FIXTURE_TEST_SUITE(reverselock_tests, BasicTestingSetup)
+#include <stdexcept>
```

Comment:
just a note: bitcoin-22086 is not partial because `policy_fee_tests.cpp` is intentionally ommited due to no Fee Filter message